seb_openedx/utils.py

```diff
@@ -0,0 +1,11 @@
+""" utils """
+import json
+
+
+def parse_json_maybe(string, default):
```

Comment:
this function gets used once. And I find that code questionable as well. If the only existence of the call goes away, this should also disapear

Reply:
Done